### PR TITLE
Devel

### PR DESCRIPTION
**Pull request by danielSoler93**

### CODE REVIEW

File: tests/data/original/test.conf [hunk 1]
```diff
@@ -0,0 +1,149 @@
+{
+  "licenseDirectoryPath" : "/scratch/PELE-repo/licenses",
+  
+  "Initialization" : {
+     "allowMissingTerminals": true,
+     "ForceField" : "OPLS2005",
+     "MultipleComplex" : [ {"files" : [{"path": "/home/dsoler/repos/frag_pele/tests/1w7h_preparation_structure_2w_carbonylC6-H7O1-C1/clustering_PDBs/1/initial_0_0.pdb" }] },
+{"files" : [{"path": "/home/dsoler/repos/frag_pele/tests/1w7h_preparation_structure_2w_carbonylC6-H7O1-C1/clustering_PDBs/1/initial_0_1.pdb" }] },
```
Comment (carlesperez94) [inline tests/data/original/test.conf:8]: Take in mind that the paths of the PDBS are from your computer.

File: setup.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 # To use a consistent encoding
 from codecs import open
 from os import path
+import frag_pele as fp
```
Comment (carlesperez94) [inline setup.py:6]: To do so we should change the __version__ in __init__.py

Comment (danielSoler93) [reply setup.py:6]: Done! I forgot to push it

File: tests/data/original/test.conf (hunk 1 above)
Comment (danielSoler93) [reply tests/data/original/test.conf:8]: True, true, I'll have a look at how to test the restart in a more general manner.

Comment (danielSoler93) [reply tests/data/original/test.conf]: The last test has been done more generally by using a templetized control file in ffad675f3653256d38ea8fc96a62d43164698872

**Pull request merged**